ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -174,11 +184,11 @@ else
 
 			param status
 
-			dmesg >> /fs/microsd/param_import_fail.txt &
+			#dmesg >> /fs/microsd/param_import_fail.txt &
```

Comment:
How does this effect other systems needs?

Reply:
whoops, removed